collection_requirements.rst

```diff
@@ -39,7 +39,7 @@ Collection Infrastructure
 =========================
 
 * MUST have a publicly available issue tracker, that does not require a paid level of service to create an account or view issues.
-* MUST use the `Ansible Code of Conduct (CoC) <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_ by copying `CODE_OF_CONDUCT.md <https://raw.githubusercontent.com/ansible-collections/.github/main/CODE_OF_CONDUCT.md>`_ into the root of your Git repository
+* MUST have a CoC (or if not fall back to Ansible's: https://docs.ansible.com/ansible/latest/community/code_of_conduct.html )
```

Comment:
I'm OK with this, though I wonder if we should add back the ache I've this by copying from collection_template